.github/workflows/main.yml

```diff
@@ -107,6 +107,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Prepare env
+        run:
+          echo "HELCIM_ENV=${{ github.ref == 'refs/heads/master' && 'production' || 'staging' }}" >> $GITHUB_ENV
```

Comment:
What about "develop" branch?

Reply:
`github.ref == 'refs/heads/master' && 'production' || 'staging'` is essentially a ternary operator, so for a non-master branch it'll be 'staging'